src/frame_clock.rs

```diff
@@ -16,4 +18,18 @@ impl FrameClock {
             (refresh_interval, presentation_time)
         }
     }
+
+    #[cfg(any(feature = "v3_8", feature = "dox"))]
+    pub fn get_current_timings(&self) -> Option<FrameTimings> {
+        unsafe {
+            from_glib_borrow(ffi::gdk_frame_clock_get_current_timings(self.to_glib_none().0))
```

Comment:
This is wrong. You must never ever return a `from_glib_borrow()` return value to safe code. It's borrowed but without making sure the owner is kept alive.

This needs to be `from_glib_none()`. Which is also what the C documentation seems to suggest, not sure why the `.gir` file says `transfer full`.

Reply:
If `from_glib_none` works then change better be done in https://github.com/gtk-rs/gir-files with fix.sh instead

Reply:
With something like
```
# fix wrong "full" transfer ownership
xmlstarlet ed -P -L \
	-u '//_:method[@c:identifier="gdk_frame_clock_get_current_timings"]/_:return-value/@transfer-ownership' -v "none" \
	-u '//_:method[@c:identifier="gdk_frame_clock_get_timings"]/_:return-value/@transfer-ownership' -v "none" \
	Gdk-3.0.gir
```